package/boot/arm-trusted-firmware-mvebu/patches/001-imagetool.patch

```diff
@@ -0,0 +1,20 @@
+--- a/plat/marvell/armada/a3k/common/a3700_common.mk
```

Comment:
It will help if this patch is sent to upstream and include it here with commit details like author, description, etc.

Reply:
Unsuitable for upstream, this is the revert described in the commit message

Reply:
Unsuitable for upstream, unsuitable for us, I guess. :-)